stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1507,6 +1510,66 @@ func _TFCOpSetAttrStringArray(_ op: CTFEOp,
   }
 }
 
+/// A TensorFlow device kind.
+public enum DeviceKind : UInt {
+  /// Default device.
+  case `default` = 0
+  /// Central processing units.
+  case cpu = 1
+  /// Graphics processing units.
+  case gpu = 2
+  // TODO: TPU?
+}
+
+internal extension _ExecutionContext {
+  func opSetDeviceFromScope(_ op: CTFEOp, _ status: CTFStatus) {
```

Comment:
Use the Swift naming convention for all  methods in the runtime. But since this is only called within the runtime entry point, why not just write all of this within that function?

We can discuss this more and see how to structure this and make it consistently swifty.

Reply:
Inlined it.